.ci/release.yml.tmpl

```diff
@@ -67,3 +67,47 @@ jobs:
             params:
               file: dist/terraform-provider-google.{{arch}}
           {% endfor %}
+
+    - name: inspec-integration-test
```

Comment:
Do we want to move this to the release pipeline or to a new `test` pipeline?

Reply:
Ah yeah, totally spaced on that. I guess that's what we want?

Reply:
I'm pretty sure that's what we want.

Reply:
IMO that will fragment our pipelines a lot, meaning that we will forget about the IT pipeline or the nightly release pipeline and never check them. But I'm happy to do it

Reply:
I'm inclined just to move it to the release pipeline, but if you have a strong preference for test pipeline, that's okay with me too.

Reply:
Yeah, that's a good point. I have no issues doing a test/release pipeline. Let's just stick with this and we can move towards 3 pipelines if people want to change.

Reply:
I'm convinced! Let's just move to release.